code/modules/reagents/Chemistry-Reagents/Chemistry-Reagents-Dispenser.dm

```diff
@@ -133,30 +184,32 @@
 		usr << "<span class='notice'>The solution dissolves the ink on the book.</span>"
 	return
 
-/datum/reagent/fluorine
-	name = "Fluorine"
-	id = "fluorine"
-	description = "A highly-reactive chemical element."
-	reagent_state = GAS
+/datum/reagent/hydrazine
+	name = "Hydrazine"
+	id = "hydrazine"
+	description = "A toxic, colorless, flammable liquid with a strong ammonia-like odor, in hydrate form."
+	taste_description = "toxin"
+	reagent_state = LIQUID
 	color = "#808080"
 
```

Comment:
Don't want this replacing anything we already have, not sure how much I care about it replacing Fluorine, though.